script/Arcueid/Day 8/08_00_ARC08_2B.txt

```diff
@@ -191,8 +191,8 @@ The insincere man leaps up and down, clutching his privates.
 -- Page 105, Offset 19932.
 -- 「底の抜けた鍋かテメエ。どう見てもそいつがトオノシキじゃねえか」
 // Italics candidate _obviously_
-"Do you have a screw loose or something, dumbass?
- This guy's %{i}obviously%{/i} Tohno Shiki."
+"Did you get dropped as a baby or something, dumbass?
+ Obviously this guy %{i}is%{/i} Tohno Shiki."
```

Comment:
Kinda prefer old version of second sentence

Reply:
Context for change:
I think putting the focus on the sentence on how obvious that Shiki is Shiki is sort of not the point, but rather that Andou has been talking to Shiki from the start. Maybe that distinction is a bit minute and unimportant, but this change was supposed to be reflect that Mario is pointing that yes, this is the guy they were looking for.

Reply:
I agree with the intent, but if I were to phrase this intent out loud I'd come up with the prev version. The new one actually feels more to me like the one you're trying to avoid - this guy obviously _is_ tohno shiki (contradicting a previous statement that he isn't) rather than this guy is _obviously_ tohno shiki (how could you not notice this it is _obvious_)

Reply:
Yeah, I've spent like 5 minutes saying it out-loud and the previous one just feels wrong to me.
The focus on obvious really bothers me.
Maybe reformulate it a bit:
"You've obviously got Tohno Shiki right in front of ya."

Reply:
Sure